unstructured/partition/pdf.py

```diff
@@ -133,9 +133,6 @@ def partition_pdf(
     """
     exactly_one(filename=filename, file=file)
 
-    if not isinstance(languages, list):
-        raise TypeError("The language parameter must be a list of language codes as strings.")
```

Comment:
Could be because type-checker complained, but the type-checker won't help when an end-user sends us a string or whatever. @Coniferish you'll need a `# type: ignore` on the `isinstance()` line if that's what it was.

Reply:
Because I moved it to `detect_languages`. That way it only has to be in one place instead of each of the partitioners. I added a test in `test_text` to check that it was being raised: `test_partition_text_element_metadata_raises_TypeError`

Reply:
Whoops. Realizing we still need it there since pdf doesn't use `detect_languages`. It was probably removed when I was removing it from other partitioners

Reply:
Actually, it's not needed there since both `pdf` and `image` rely on `partition_pdf_or_image` and it's handled there. I'll change the message, though, to add an example

Reply:
Made a commit to remove redundant error catches and updated this message